adal/src/androidTest/java/com/microsoft/aad/adal/DiscoveryTests.java

```diff
@@ -384,23 +384,6 @@ public void testValidateAuthorityFailedWithoutNetwork() throws IOException {
         }
     }
 
-    @Test
```

Comment:
This test is now invalid, due to doze mode detection changes that now use the internet connection if active/available irrespective of if the current app thinks its in doze mode